library/src/main/res/values/colors.xml

```diff
@@ -20,5 +20,7 @@
     <color name="chucker_status_300">#0D47A1</color>
 
     <color name="chucker_background_span_color">#ffffff00</color>
-    <color name="chucker_foreground_span_color">#ffff0000</color>
+    <color name="chucker_foreground_span_color">#FF5900</color>
```

Comment:
Was this intentional?

Reply:
Sorry, that was not intentional.

Reply:
sorry, that was not intentional hence reverted.